crates/autopilot/src/database/recent_settlements.rs

```diff
@@ -1,19 +1,19 @@
 use {anyhow::Context, primitive_types::H256};
 
 impl super::Postgres {
-    pub async fn find_tx_hash_by_auction_id(
-        &self,
-        auction_id: i64,
-    ) -> anyhow::Result<Option<H256>> {
+    pub async fn find_settlement_transactions(&self, auction_id: i64) -> anyhow::Result<Vec<H256>> {
```

Comment:
Looks like this function might as well be inlined in the `infra` module to get rid of the file.